src/components/Landing/QAs.tsx

```diff
@@ -19,32 +34,74 @@ export default function QAs() {
     {
       q: t`What does Juicebox cost?`,
       a: [
-        t`Juicebox is an open protocol on Ethereum that is funded using Juicebox itself. You can check out the contractualized budget specs at https://juicebox.money/#/p/juicebox.`,
-        t`5% of all money made by projects is sent to help pay for the maintenance and development of Juicebox itself. In exchange, projects get Juicebox tokens ($JBX), which will be worth more as the ecosystem grows over time.`,
+        <Trans>
+          Juicebox is an open protocol on Ethereum that is funded using Juicebox
+          itself. You can check out the contractualized budget specs{' '}
+          <a
+            href="https://juicebox.money/#/p/juicebox"
+            rel="noreferrer"
+            target="_blank"
+          >
+            {' '}
+            here
+          </a>
+          .
+        </Trans>,
+        <Trans>
+          Projects building on Juicebox pay a 5% JBX membership fee from
+          withdrawn funds into the JuiceboxDAO treasury. Projects can then use
+          their JBX to participate in the governance of JuiceboxDAO, as well as
+          redeem from its growing {overflowVideoLink('overflow')}. The fee is
```

Comment:
**suggestion**: can we render this link with JSX?

`<OverflowVideoLink type='overflow' />`

Reply:
Yep good call